### ISSUE TITLE
[Bug]: Series Sequence Info added to Series tag when book is readded

### ISSUE THREAD

**Issue by ceramicwhite** [advplyr/audiobookshelf#1360]
### Describe the issue

If a book has been removed from ABS and then imported again with the `metadata.abs`, the Series and Sequence are concatenated on import.

```
;ABMETADATA2
....
series=Mountain Man #1-3
```
![Screenshot 2023-01-04 121351](https://user-images.githubusercontent.com/7660824/210642966-a35e7431-3ec0-43fa-91af-ac729c52102d.png)

### Steps to reproduce the issue

1. Remove book that's part of a series and has a sequence and no embedded ID3 tags (settings dependent)
2. Scan Library to reimport book (note I store `metadata.abs` with titles)

### Audiobookshelf version

v2.2.11

### How are you running audiobookshelf?

Docker

**Comment by ceramicwhite** [comment]
@advplyr I tried master and didn't seem to fix the problem. Also after removing the book if I do a scan of library it gets cancelled on master:
`DEBUG [Server] Cancel scan lib_.....`

**Comment by advplyr** [comment]
I just tested and it is working for me.

![image](https://user-images.githubusercontent.com/67830747/211115317-c77034ac-36df-4f9b-b218-a52922d08371.png)

For this test I went and edited the `metadata.abs` in one of my book folders. 

```
;ABMETADATA2
#audiobookshelf v2.2.11

media=book
title=A Test Book
subtitle=
authors=Test Author
narrators=
publishedYear=2019
publisher=
isbn=
asin=
language=
genres=Fantasy
series=Mountain Man #1-3
explicit=N
```

After saving that I go press edit on the book in Abs and press the "Re-Scan" button. The series gets updated correctly.

**Comment by ceramicwhite** [comment]
@advplyr Oh I was looking at some other sequences I had:
`series=Star Wars #39 Kids`
`series=Expeditionary Force #10.8 - Mavericks: Book 2`

 I think Series, Collection, & Playlist could potentially use a rework.

I lean heavily on the sequence part to organize a series.
I think that series should have sub-sequences that allow for ordering by factors such as publication date or reading order.
couple examples:
![enderverse](https://user-images.githubusercontent.com/7660824/211121455-07439d5e-313a-4872-80fe-ec5b68ab17e6.png)
![expeditionary-force](https://user-images.githubusercontent.com/7660824/211121470-fe95807c-4cf0-4ae5-a094-05b9d12f4a73.png)
![wakingup](https://user-images.githubusercontent.com/7660824/211121496-23dcdf56-d4af-4084-8030-74d978d6ea30.png)

This last one should be a collection, but I'm not a fan of the layout for collections. Playlists have the same view as series, so I would prefer to switch the styles of collections and playlists. Currently collections view makes more sense for playlists and the series style makes more sense for collections imo.
![wwii](https://user-images.githubusercontent.com/7660824/211121537-97bcdac8-2163-4c69-8a9b-8ded9ebb4d21.png)

**Comment by advplyr** [comment]
I don't think those long strings should be used for series sequence. I think Audible enforces the sequence to be a number, but I thought we should allow some alphanumeric characters just in case.

It seems more like you are asking for a sort option on the series page. I'm not really sure what specifically you are trying to do.

**Comment by advplyr** [comment]
Fixed in [v2.2.12](https://github.com/advplyr/audiobookshelf/releases/tag/v2.2.12)
